components/HomeHero/Strapline/Strapline.module.scss

```
@@ -0,0 +1,31 @@
+.strapline {
+    @include font-size(m);
+    display: flex;
+    line-height: 1.5;
+    flex-direction: column;
```

Minor nitpick - I tend to group related CSS rules together, i.e. 
```suggestion
    display: flex;
    flex-direction: column;
    line-height: 1.5;
```
It's nicer for reading the code